webapp/app/[locale]/stake/dashboard/_components/stakeAssetsTable/welcomeStake.tsx

```diff
@@ -0,0 +1,122 @@
+import { Card } from 'components/card'
+import { useTranslations } from 'next-intl'
+
+import { StakeAndEarnPointsGraphIcon } from '../../../_components/icons/stakeAndEarnPointsGraph'
+
+interface Props {
+  onClick: VoidFunction
+}
+
+interface StarIconProps {
+  className: string
+  color?: string
+  rotation?: number
+  size?: number
+}
+
+const StarIcon = ({
+  className,
+  color = '#FF6014',
+  rotation = 0,
+  size = 14,
+}: StarIconProps) => (
+  <svg
+    className={className}
+    fill={color}
+    height={size}
+    style={{
+      transform: `rotate(${rotation}deg)`,
+    }}
```

Comment:
```suggestion
    {...( rotation !== 0 ? { style: { transform: `rotate(${rotation}deg)` } } : undefined )}
```

I think this works better, so we don't add an unnecessary rotation styling when it isn't needed. You could also remove the default value `0` and check `rotation` against undefined.